packages/jellyfish-api-core/__tests__/category/account/getAccountHistory.test.ts

```diff
@@ -0,0 +1,99 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { Testing } from '@defichain/jellyfish-testing'
+import waitForExpect from 'wait-for-expect'
+import { RpcApiError } from '@defichain/jellyfish-api-core'
+
+describe('Account', () => {
+  const testing = Testing.create(new MasterNodeRegTestContainer())
```

Comment:
would be great if https://github.com/DeFiCh/jellyfish/pull/1035 can be merged before this and use it.

Reply:
@chanakasameera https://github.com/DeFiCh/jellyfish/pull/1035#discussion_r808761851